gammapy/utils/coordinates/fov.py

```diff
@@ -1,122 +1,192 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import absolute_import, division, print_function, unicode_literals
 import numpy as np
+from astropy.coordinates import SkyCoord, SkyOffsetFrame
 
 __all__ = ["fov_to_sky", "sky_to_fov"]
 
-
 def fov_to_sky(lon, lat, lon_pnt, lat_pnt):
     """Make a transformation from field-of-view coordinates to sky coordinates.
 
     Parameters
     ----------
-    lon : array_like
-        Field-of-view longitude coordinate to be transformed, in degrees
-    lat : array_like
-        Field-of-view latitude coordinate to be transformed, in degrees
-    lon_pnt : array_like
-        Longitude coordinate of the pointing position, in degrees
-    lat_pnt : array_like
-        Latitude coordinate of the pointing position, in degrees
+    lon : `~astropy.units.Quantity`
```

Comment:
You can write `lon, lat :`, i.e. group doctring parameters that have the same type and are easily described together. Save a few lines.